packages/wdio-cli/src/constants.js

```diff
@@ -114,6 +115,45 @@ export const QUESTIONNAIRE = [{
     message: 'What is the port on which that service is running?',
     default: '80',
     when: /* istanbul ignore next */ (answers) => answers.backend.indexOf('different service') > -1
+}, {
+    type: 'input',
+    name: 'exp_env_access_key',
+    message: 'Access key from Experitest Cloud',
+    default: 'EXPERITEST_ACCESS_KEY',
+    when: /* istanbul ignore next */ (answers) => answers.backend === 'In the Experitest cloud'
+}, {
+    type: 'input',
+    name: 'exp_env_hostname',
+    message: 'Environment variable for cloud url',
+    default: 'example.experitest.com',
+    when: /* istanbul ignore next */ (answers) => answers.backend === 'In the Experitest cloud'
+}, {
+    type: 'list',
+    name: 'exp_env_port',
+    message: 'Choose a port for environment variable',
+    default: 443,
+    choices: [
+        443,
+        80,
+        'Other'
+    ],
+    when: /* istanbul ignore next */ (answers) => answers.backend === 'In the Experitest cloud'
+}, {
+    type: 'input',
+    name: 'exp_env_port',
+    message: 'Environment variable for other port',
```

Comment:
Does Experitest allow multiple ports to be set? If not we don't need to ask this.

Reply:
Only a single port is allowed but it's can be configurable by the customer to be any port.